test/support/runPlaywrightTests.js

```diff
@@ -33,8 +35,20 @@ const runTests = async (browserType) => {
     });
 
     // Expose a function inside the playwright browser to exit with the right status code when tests pass/fail
-    page.exposeFunction('onTestResult', ({ detail }) => {
+    page.exposeFunction('onTestResult', ({ detail, jUnitReport }) => {
```

Comment:
remove `jUnitReport` from destructured properties, it seems it's only present in `detail` (based on playwrightSetup.js)